src/components/Calendar.tsx

```diff
@@ -50,11 +53,11 @@ export function Calendar({
     timeEnd,
   ]);
 
-  const goForward = useCallback(() => setFocusDate(view === 'day' ? addDays(1) : addWeeks(1)), [
+  const goForward = useCallback(() => setFocusDate(view === 'week' ? addWeeks(1) : addDays(1)), [
```

Comment:
I imagine these functions to work differently. Since you switched the statement, I suppose that the current behaviour is intentional. I would personally expect that it would move the whole view period by the amount of custom days. But I am happy to discuss this. This is a line that works (even though it is not very readable):
```ts
const goForward = useCallback(
  () => setFocusDate(view === 'week' ? addWeeks(1) : addDays(view === 'day' ? 1 : customDays)),
  [view]
);
```

Reply:
If we change this, could you also change line 192 in the example's `index.tsx`?
```ts
if (view !== 'day') {
  date = format(viewPeriod.start, 'MMM d') + ' - ' + format(viewPeriod.end, 'MMM d');
}
```

Reply:
I see what you mean, weirdly I prefer the other way but I am happy to discuss about it when we have the chance :)

Reply:
I think this would be more in line with the behaviour of the day and week views. What if we would change the signature of `setDate` to:

```ts
setDate: (date: Date | (Date => Date)) => void;
```

Only the TS definition would have to be adopted. This would allow you to easily implement the forward and backward one day functions.

```ts
import { addDays, subDays } from 'date-fns/fp';

const { setDate } = useCalendar();
const goForwardOneDay = setDate(addDays(1));
const goBackwardOneDay = setDate(subDays(1));
```

Reply:
ok got it!

Reply:
I couldn't make it work, `addDays` was a bit tricky. so I left it with customDays